spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
they are separate tests. they are asserting their own preconditions. factoring out the clear statement would bring the required state outside of the local scope and will be missed when more specs are added in future, or when that precondition is not required for the series of tests. yes, the effect is the same, but the confusion here is for the programmer not the program.

create also cannot be used, because the records must be created before visiting the page. this is one of the problems of using before statements.